src/coreComponents/denseLinearAlgebra/interfaces/blaslapack/BlasLapackLA.hpp

```diff
@@ -443,6 +443,25 @@ struct BlasLapackLA
   static void matrixInverse( MatColMajor< real64 const > const & A,
                              MatColMajor< real64 > const & Ainv );
 
+  /**
+   * @brief Solves the linear system ;
+   * \p Ainv  \p solution = \p rhs.
```

Comment:
Can you elaborate a little the method used during resolution?
And what is `Ainv` in your documentation? The inverse of `A`? Is it intended?

Reply:
no, I don't think it should be `Ainv`. I am not sure how that showed up. I thought this would evolve into a generic interface that can use various libraries soI would not add too many details about the resolution method (also coz I am not sure what lapack exactly does. Probably a simple gaussian elimination). Maybe we can say it s meant for small dense linear systems.